packages/stacked_cli/lib/src/services/analytics_service.dart

```diff
@@ -1,66 +1,97 @@
 import 'package:usage/usage_io.dart';
 
+/// Provides functionality to interact with Google Analytics
 class AnalyticsService {
   final AnalyticsIO _analytics = AnalyticsIO(
     'UA-41171112-5',
     'stacked-cli',
     'stacked_cli',
   );
 
-  Future<void> appCreated({required String name}) async {
+  /// This will wait until all outstanding analytics requests have completed,
+  /// or until the specified duration has elapsed.
+  Future<void> waitLastPingOrCloseAtTimeout() async {
+    await _analytics.waitForLastPing(
```

Comment:
Good catch. I didn't know about this. I hope this doesn't mean that we could potentially lose some analytics.